gpkit/interactive/sensitivity_map.py

```diff
@@ -147,13 +150,19 @@ def constraint_latex_list(self, paintby):
             idx = len(self.model.cost.exps) if paintby == "monomials" else 1
             senss = sol["sensitivities"][paintby][idx:]
         colorfn = colorfn_gen(max(senss))
-
+        tup_list = []
         for i, constr in enumerate(self.model):
             if self.paintby == "constants":
                 left = signomial_print(constr.left, sol, colorfn, paintby)
                 right = signomial_print(constr.right, sol, colorfn, paintby)
                 constr_tex = ("    & %s \\\\" %
-                              (left + constr.latex_opers[constr.oper] + right))
+                              (left[0] + constr.latex_opers[constr.oper]
+                               + right[0]))
+                for j in right[1]+left[1]:
+                    num_tup = (j[6:10], j[11:15], j[16:-1])
+                    if num_tup != ('0.70', '0.70', '0.70') \
+                                  and num_tup not in tup_list:
+                        tup_list.append(num_tup)
```

Comment:
I'm not sure I follow this. Is there a simpler / cleaner way to accomplish this?